Epam.TestAutomation.Tests/BaseTest.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Epam.TestAutomation.Core.Browser;
+using Epam.TestAutomation.Core.Helper;
+using Epam.TestAutomation.Core.Utils;
+using NUnit.Framework;
+using OpenQA.Selenium.Interactions;
+using Logger = Epam.TestAutomation.Core.Utils.Logger;
+
+namespace Epam.TestAutomation.Tests;
+
+public abstract class BaseTest
+{
+    public TestContext TestContext { get; set; }
+    public Actions action { get;}
```

Comment:
Remove Action property as you already have realization of Actions class in Browser class.